[2025-10-11T06:25:42.929+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-11T06:25:42.999+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipeline_flow.check_create_stg_tables scheduled__2000-02-06T00:00:00+00:00 [queued]>
[2025-10-11T06:25:43.019+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipeline_flow.check_create_stg_tables scheduled__2000-02-06T00:00:00+00:00 [queued]>
[2025-10-11T06:25:43.021+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-10-11T06:25:43.233+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): check_create_stg_tables> on 2000-02-06 00:00:00+00:00
[2025-10-11T06:25:43.253+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62 DeprecationWarning: This process (pid=12553) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-10-11T06:25:43.256+0000] {standard_task_runner.py:64} INFO - Started process 12568 to run task
[2025-10-11T06:25:43.253+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'pipeline_flow', 'check_create_stg_tables', 'scheduled__2000-02-06T00:00:00+00:00', '--job-id', '1460', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_dag.py', '--cfg-path', '/tmp/tmpd4q_fosi']
[2025-10-11T06:25:43.259+0000] {standard_task_runner.py:91} INFO - Job 1460: Subtask check_create_stg_tables
[2025-10-11T06:25:43.397+0000] {task_command.py:426} INFO - Running <TaskInstance: pipeline_flow.check_create_stg_tables scheduled__2000-02-06T00:00:00+00:00 [running]> on host 321597a50810
[2025-10-11T06:25:43.626+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='oluwaseyiogundimu10@gmail.com' AIRFLOW_CTX_DAG_OWNER='seyi' AIRFLOW_CTX_DAG_ID='pipeline_flow' AIRFLOW_CTX_TASK_ID='check_create_stg_tables' AIRFLOW_CTX_EXECUTION_DATE='2000-02-06T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2000-02-06T00:00:00+00:00'
[2025-10-11T06:25:43.631+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-11T06:25:43.667+0000] {connection.py:22} INFO - Successfully connected to Postgres.
[2025-10-11T06:25:43.805+0000] {connection.py:43} INFO - Table 'stg_users' in Postgres.
[2025-10-11T06:25:43.809+0000] {connection.py:22} INFO - Successfully connected to Postgres.
[2025-10-11T06:25:43.964+0000] {connection.py:43} INFO - Table 'users' in Postgres.
[2025-10-11T06:25:43.966+0000] {connection.py:22} INFO - Successfully connected to Postgres.
[2025-10-11T06:25:44.093+0000] {connection.py:43} INFO - Table 'stg_movies' in Postgres.
[2025-10-11T06:25:44.095+0000] {connection.py:22} INFO - Successfully connected to Postgres.
[2025-10-11T06:25:44.208+0000] {connection.py:43} INFO - Table 'movies' in Postgres.
[2025-10-11T06:25:44.210+0000] {connection.py:22} INFO - Successfully connected to Postgres.
[2025-10-11T06:25:44.323+0000] {connection.py:43} INFO - Table 'stg_ratings' in Postgres.
[2025-10-11T06:25:44.325+0000] {connection.py:22} INFO - Successfully connected to Postgres.
[2025-10-11T06:25:44.430+0000] {connection.py:43} INFO - Table 'ratings' in Postgres.
[2025-10-11T06:25:44.431+0000] {load.py:32} INFO - Tables exist.
[2025-10-11T06:25:44.433+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-10-11T06:25:44.435+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-11T06:25:44.458+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=pipeline_flow, task_id=check_create_stg_tables, run_id=scheduled__2000-02-06T00:00:00+00:00, execution_date=20000206T000000, start_date=20251011T062543, end_date=20251011T062544
[2025-10-11T06:25:44.518+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-10-11T06:25:44.578+0000] {taskinstance.py:3503} INFO - 3 downstream tasks scheduled from follow-on schedule check
[2025-10-11T06:25:44.582+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
